js.go

```diff
@@ -267,16 +267,16 @@ const (
 	TraceReceived TraceOperation = 1
 )
 
-// TraceCB is called to trace API interactions for the JetStream Context
-type TraceCB func(op TraceOperation, subj string, payload []byte, hdr Header)
+// ClientTrace can be used to trace API interactions for the JetStream Context.
+type ClientTrace struct {
+	RequestSent      func(subj string, payload []byte, hdr Header)
```

Comment:
Well, since we always call RequestSent() with nil as the last argument, why make this function accept hdr? RequestSent and RequestReceived don't have to have the same signature, right?

Reply:
Good point, not needed for requests to the JS API so removed the last parameter.